operator/README.md

```diff
@@ -56,6 +56,12 @@ NAME                               READY   STATUS    RESTARTS   AGE
 karmada-operator-5b7f485c5-g5lj5   1/1     Running   0          26s
 ```
 
+### Install karmada crds
```

Comment:
```suggestion
### Install Karmada operator crds
```

Reply:
@liangyuanpeng thanks. It should be `Karmada operator`.

Reply:
Updated.